apps/omg_watcher/test/omg_watcher/integration/block_getter_test.exs

```diff
@@ -215,4 +215,34 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
     # we should still be able to challenge this "unchallenged exit" - just smoke testing the endpoint, details elsewhere
     WatcherHelper.get_exit_challenge(exit_blknum, 0, 0)
   end
+
+  @tag fixtures: [:in_beam_watcher, :mix_based_child_chain, :test_server, :stable_alice, :stable_alice_deposits]
```

Comment:
comment about the next line actually, putting here to not pollute thread :point_down: 

> test "operator claim more fees than collected", %{

(also about the comment saying)

>  that overclaim fees

Name and comment are accurate, but they do not convey that this holds for any problem with the fee txs. I think we can say that this intends to test operator's fee claiming being incorrect in general.

But not a big deal anyway.